openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx

```diff
@@ -202,10 +238,57 @@ const MlModelPage = () => {
     }
   };
 
+  const getEntityFeedCount = () => {
```

Comment:
we are not returning anything so name should be `fetchEntityFeedCount` instead `getEntityFeedCount`